tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -173,53 +173,45 @@ void check_vectors(const T1& t1, const T2& t2) {
 
 SPECTRE_TEST_CASE("Unit.DataStructures.DataVector.MathAfterMove",
                   "[Unit][DataStructures]") {
```

Comment:
As Will pointed out, things that are seemingly redundant may not be with DataVector. The internals are keeping track of owning-ness in non-trivial ways and so altering the tests is something I frown upon. A fair number of the tests were added as a result of bugs and so altering them could allow re-introduction of said bug.